contracts/contracts/offer/src/contract.rs

```diff
@@ -162,13 +170,25 @@ pub fn update_offer(
 ) -> Result<Response, ContractError> {
     assert_min_g_max(msg.min_amount, msg.max_amount)?;
 
+    let hub_config = get_hub_config(deps.as_ref());
     let mut offer_model = OfferModel::may_load(deps.storage, &msg.id);
 
-    assert_ownership(info.sender, offer_model.offer.owner.clone())?;
+    assert_ownership(info.sender.clone(), offer_model.offer.owner.clone())?;
+
+    let mut send_msgs: Vec<SubMsg> = Vec::new();
```

Comment:
I think that this should be better named as `sub_msgs` or something similar. 

imo, this name (`send_msgs`) would better represent a vec of `Bank::Send` msgs, like we have in `release_escrow`, etc.